services/greasyfork/greasyfork-rating.service.js

```diff
@@ -0,0 +1,132 @@
+import { floorCount as floorCountColor } from '../color-formatters.js'
```

Comment:
(Speaking to this entire class, not specific to this one line)

I'm not sure how much value there is in a set of badges that just display the counts, but perhaps I'm just not familiar enough with the service.

What comes to mind for me as far as existing badges we provide that have some level of similarity are ones like VS/Open VSX rating and pkgreview where we display the rating (star count), optionally with the quantity of reviews as a suffix. Would something like that even be feasible with Greasy Fork or are the actual rating values/aggregated rating values not available through the API?

Reply:
Greasy Fork doesn't really have an aggregated score or star system like most platforms.

This is how ratings are displayed on their website, just counts for Good, OK, and Bad ratings. When rating scripts, you select one of these 3 categories.

![image](https://user-images.githubusercontent.com/20955511/173486058-6e0c5e36-cd08-4e2a-97bd-147cbb3ad320.png)

Reply:
Thanks for sharing, very helpful. I think it would be best to at least pull out the rating badges from this PR so we can go ahead and get the rest of them merged, and then circle back to if/when/how on the ratings badges